ocsci/tests/test_config.py

```diff
@@ -16,4 +16,3 @@ def test_defaults(self):
         config_sections = ocsci.config.to_dict().keys()
         for section_name in config_sections:
             section = getattr(ocsci.config, section_name)
-            assert section == getattr(ocs.defaults, section_name)
```

Comment:
This makes the test do nothing at all. Why does it need to be changed?

Reply:
that was old change, just updated it now after separating out defaults.

Reply:
I'm really confused - your new commits are removing defaults from defaults.py now

Reply:
some where duplicated in defaults and ENV_DATA, I only removed those duplicates.